plotly/Cargo.toml

```diff
@@ -18,9 +18,11 @@ kaleido = ["plotly_kaleido"]
 plotly_ndarray = ["ndarray"]
 plotly_image = ["image"]
 wasm = ["getrandom", "js-sys", "wasm-bindgen", "wasm-bindgen-futures"]
+with-axum = ["askama/with-axum", "askama_axum"]
```

Comment:
Maybe the feature name could be a bit better, but naming is always hard :grin: 
Will merge it like this for now.